### ISSUE TITLE
Better display for dark favicons in the dark theme

### ISSUE THREAD

**Issue by sabi0** [Croydon/vertical-tabs-reloaded#114]
Dakr favicons (e.g. GitHub's one) are poorly visible when using dark theme. They should be inversed or highlighted in some other way.

Here is an example how they did it in Opera:
https://blogs.opera.com/desktop/2017/06/opera-47-beta/

**Comment by Croydon** [comment]
Does someone know a JavaScript library which can do the entire image manipulation?

I won't have time to write something on my own for likely a long time.

Integrating something existing should be fairly easy.

A library should somehow calculate the brightness of the entire icon, then decide if the colors needs to get inversed. Or something along these lines.

**Comment by Croydon** [comment]
This will finally land in the 0.13.0 release that I'm planning for the next few days

I might improve it in later releases further, see #224

**Comment by Croydon** [comment]
Closing.

Fixed by e901efcb97187a228d30dac249c659b4b7ed3359, dda4587c7af0a7ad777ebd3739eefc2ed6e45938, 7f8d4d3ff52e927aac68ec81283d88be91de958a 

Thanks @mmktomato!
